api/src/pcapi/local_providers/cinema_providers/cds/cds_stocks.py

```diff
@@ -8,6 +11,8 @@
 from pcapi.core.external_bookings.models import Movie
 from pcapi.core.offerers.models import Venue
 from pcapi.core.offers.models import Offer
+from pcapi.core.offers.models import PriceCategory
+from pcapi.core.offers.models import PriceCategoryLabel
 from pcapi.core.offers.models import Product
 from pcapi.core.offers.models import Stock
```

Comment:
Si tu as le courage pour un BSR de ré-écriture de ces imports en 
```suggestion
import pcapi.core.offers.models as offers_models
```
Ce serait merveilleux !